README.md

```diff
@@ -0,0 +1,134 @@
+<p align="center">
+  <a href="https://rudderstack.com/">
+    <img alt="RudderStack" width="512" src="https://raw.githubusercontent.com/rudderlabs/rudder-sdk-js/develop/assets/rs-logo-full-light.jpg">
+  </a>
+  <br />
+  <caption>The Customer Data Platform for Developers</caption>
+</p>
+<p align="center">
+  <b>
+    <a href="https://rudderstack.com">Website</a>
+    ·
+    <a href="https://rudderstack.com/docs/">Documentation</a>
+    ·
+    <a href="https://rudderstack.com/join-rudderstack-slack-community">Community Slack</a>
+  </b>
+</p>
+
+---
+
+# RudderStack Swift SDK
+
+The Swift SDK enables you to track customer event data from your iOS, macOS, tvOS, and watchOS applications and send it to your configured destinations via RudderStack.
+
+## Table of Contents
+
+- [Installing the Swift SDK](#installing-the-swift-sdk)
+- [Initializing the SDK](#initializing-the-sdk)
+- [Identifying users](#identifying-users)
+- [Tracking user actions](#tracking-user-actions)
+- [Contact us](#contact-us)
+- [Follow Us](#follow-us)
+
+---
+
+## Installing the Swift SDK
+
+### Swift Package Manager
+
+Add the SDK to your Swift project using Swift Package Manager:
+
+1. In Xcode, go to `File > Add Package Dependencies`
+2. Enter the repository URL: `https://github.com/rudderlabs/rudder-sdk-swift`
+3. Select the version you want to use
```

Comment:
Let's add some screenshots for clarity: Refer to iOS-v1 instructions on how to add SDK using SPM https://github.com/rudderlabs/rudder-sdk-ios.

I think we also need to select the target while adding the SDK: 
<img width="645" alt="image" src="https://github.com/user-attachments/assets/29800671-8e37-46ac-b037-02cd67dab2ed" />
Please add this step along with the screenshot.